code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -14,6 +14,9 @@
 		SHC = (for(components of recipe) total_SHC *= component SHC)
 
 
+
+	It is very common to use REAGENTS_METABOLISM or REM used in this file. Both are the same value, 0.5
```

Comment:
I think you mean `REAGENTS_EFFECT_MULTIPLIER` and `REM` are equivalent. `REAGENTS_METABOLISM` is not.